client/src/components/LoginPage/index.js

```diff
@@ -1,86 +1,110 @@
 import React, { useState } from "react";
-import { Button, makeStyles } from "@material-ui/core";
-import TextField from "@material-ui/core/TextField";
+import { Button } from "@material-ui/core";
+import UserContext from "../../UserContext";
 import Footer from "./Footer";
+import InputField from "./InputField";
 import "./Login.css";
+import Dialog from "../shared/Dialog";
+import axios from "axios";
 
-const styles = makeStyles({
-  textBox: {
-    margin: 10,
-    width: "80%",
-  },
-});
-
-// const [state name, function to update state]
-// body = default state
 const LoginPage = (props) => {
-  const [user, setTempUser] = useState({
+  const { user, setUser } = React.useContext(UserContext);
+  const [emailError, setEmailError] = useState(false);
+  const [passwordError, setPasswordError] = useState(false);
+  const [error, setError] = useState(false);
+  const [temp, setTempUser] = useState({
```

Comment:
Consider naming conventions here.  You have 'setTempUser' but the associated state variable is just "temp".  All of your other state variables / setters use correct conventions though so good job.

Reply:
yeah I was meaning to do so - good call